cmake/FindFJContrib.cmake

```diff
@@ -0,0 +1,25 @@
+# * Locate FastJetContrib library Defines:
```

Comment:
It would be nice if we could upstream the cmake support so it's in the next fastjet and fjcontrib releases. Then we don't have to maintain FindFastJet.cmake and FindFJContrib.cmake.

Reply:
For my own understanding (since I'm very unfamiliar with cmake), would this entail adding FastJet and FJContrib to the top-level CMakeLists?

Reply:
No need to add to the top-level CMakeLists. I meant contributing FindFastJet.cmake and FindFJContrib.cmake as PRs to the fastjet project. These are files that not just us are using, but likely many others too. I suspect you copied them from another project in the first place. Having fastjet provide the reference implementation will ensure uniformity.

Reply:
Ah! I see! I have to admit, I haven't ever used them elsewhere since all other projects I've worked on didn't use cmake 😅 

But I agree! That would be a better approach long-term!

Reply:
I'm not sure that we can contribute cmake support directly. The fastjet.fr website doesn't mention any route to do that. We can contact the developers to see if they are willing to accept external contributions.